src/graphql/queries.js

```diff
@@ -415,6 +418,82 @@ module.exports = {
     return cursor.exec();
   },
 
+  mostVotes: async (root, { filter, orderBy, limit, skip }, { db: { Votes } }) => {
+    const voterFilters = buildVoteFilters(filter);
+    voterFilters.forEach((element) => {
+      if (element.topicAddress == null) {
+        throw new Error('topicAddress is not passed in');
+      }
+    });
+    const query = filter ? { $or: voterFilters } : {};
+    const cursor = Votes.cfind(query);
+    const result = await cursor.exec();
+
+    const accumulated = result.reduce((acc, cur) => {
+      if (acc.hasOwnProperty(cur.voterAddress)) {
+        acc[cur.voterAddress] += Number(cur.amount);
+      } else {
+        acc[cur.voterAddress] = Number(cur.amount);
+      }
+      return acc;
+    }, {});
+
+    let votes = Object.keys(accumulated).map(key => ({ voterAddress: key, amount: String(accumulated[key]), topicAddress: voterFilters[0].topicAddress }));
+    votes.sort((a, b) => b.amount - a.amount);
+
+    const totalCount = votes.length;
+    const ret = { totalCount, votes };
+
+    if (_.isNumber(limit) && _.isNumber(skip)) {
+      const end = skip + limit;
+      const hasNextPage = end < totalCount;
+      votes = votes.splice(skip, end);
+      const pageNumber = _.toInteger(end / limit); // just in case manually enter not start with new page, ex. limit 20, skip 2
+      ret.pageInfo = {
+        hasNextPage,
+        pageNumber,
+        count: votes.length,
+      };
+      ret.votes = votes;
+    }
+
+    return ret;
+  },
+
+  winners: async (root, { filter, orderBy, limit, skip }, { db: { Votes } }) => {
+    const voterFilters = buildVoteFilters(filter);
+    const query = filter ? { $or: voterFilters } : {};
+    const result = await Votes.find(query); //get all winning votes
+    const filtered = [];
+    _.each(result, (vote) => {
+      if (!_.find(filtered, {
+        voterAddress: vote.voterAddress,
+        topicAddress: vote.topicAddress,
+      })) {
+        filtered.push(vote);
+      }
+    });
+    let winnings = [];
+
+    for (const item of filtered) {
+      winnings.push( await (a(item)));
+    }
+    winnings = _.orderBy(winnings,['amount'], ['desc'] )
+    async function a (item)  {
```

Comment:
couple of comments:
1. either remove this function entirely and just call `TopicEvent.calculateWinnings()` as the API does in the for loop. or move this function outside of the `module.exports` so it can be reused.
2. it's good practice to name it something that you can tell without having to read the code like `getWinnings()`